tests/unit/lms/tasks/email_digests_test.py

```diff
@@ -223,148 +224,52 @@ def send_instructor_email_digest(self, patch):
 
 @pytest.mark.usefixtures("digest_service")
 class TestSendInstructorEmailDigests:
-    def test_it(
-        self, created_before, digest_service, db_session, h_userid, make_task_done
-    ):
-        # The TaskDone that we expect get_task_done() to return.
-        matching_task_done = make_task_done(created_before=created_before.isoformat())
-
-        db_session.add_all(
-            [
-                # An older TaskDone for the same user, this should be ignored.
-                make_task_done(
-                    created_before=(created_before - timedelta(days=1)).isoformat()
-                ),
-                # A TaskDone with a different type, this should be ignored.
-                make_task_done(type="DIFFERENT"),
-                # A TaskDone with a different h_userid, this should be ignored.
-                make_task_done(h_userid="DIFFERENT"),
-                # A TaskDone with None in the data column, this should be ignored.
-                factories.TaskDone(data=None),
-                # A TaskDone with no h_userid, this should be ignored.
-                make_task_done(omit=["h_userid"]),
-                # A TaskDone with no created_before, this should be ignored.
-                make_task_done(omit=["created_before"]),
-                # A TaskDone with an invalid created_before, this should be ignored.
-                make_task_done(created_before="foo"),
-                # The TaskDone that we expect get_task_done() to return.
-                matching_task_done,
-            ]
-        )
```

Comment:
No point in have a test that fills the DB with `task_done`'s now that the code no longer reads that table